MHacks/AnnouncementsViewController.swift

```diff
@@ -97,16 +97,21 @@ class AnnouncementsViewController: UITableViewController {
     }
 	
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        
         let cell = tableView.dequeueReusableCell(withIdentifier: "item", for: indexPath) as! AnnouncementCell
-        
+
         let announcement = APIManager.shared.announcements[(indexPath as NSIndexPath).row]
-        
+		
         cell.titleLabel.text = announcement.title
         cell.dateLabel.text = announcement.localizedDate
         cell.messageLabel.text = announcement.message
 		cell.colorView.layer.borderColor = announcement.category.color.cgColor
 		cell.colorView.layer.borderWidth = cell.colorView.frame.width
+		
+		if !announcement.isSponsored {
+			cell.contentStackView.removeArrangedSubview(cell.sponsoredLabel)
+			cell.sponsoredLabel.removeFromSuperview()
```

Comment:
This is fine but you could more simply accomplish the same affect by setting the hidden property. UIStackView will correctly not allocate space for hidden views.